providers/implementations/kem/build.info

```diff
@@ -12,3 +13,6 @@ IF[{- !$disabled{ec} -}]
     SOURCE[$EC_KEM_GOAL]=ecx_kem.c
   ENDIF
 ENDIF
+
+SOURCE[$TEMPLATE_KEM_GOAL]=template_kem.c
+INCLUDE[$TEMPLATE_KEM_GOAL]=../../common/include ../../implementations/include ../../fips/include ../../../include
```

Comment:
I would put the INCLUDE[] into providers/build.info like other libs have so it doesn't have to be repeated in all subdirectories.

Reply:
good point. so changed.

Reply:
ok to mark resolved @t8m?